server/form-pages/apply/risks-and-needs/risk-to-self/historicalRisk.ts

```diff
@@ -21,6 +22,12 @@ export default class HistoricalRisk implements TaskListPage {
     },
   }
 
+  importDate = this.application.data?.['risk-to-self']?.['historical-risk'].dateOfOasysImport
```

Comment:
I wonder if it would be good to have a function to handle this as I am finding it a bit hard to read...